xapian-letor/api/featurelist.cc

```diff
@@ -98,7 +98,7 @@ FeatureList::create_feature_vectors(const Xapian::MSet & mset,
 				    const Xapian::Database & letor_db) const
 {
     LOGCALL(API, std::vector<FeatureVector>, "FeatureList::create_feature_vectors", mset | letor_query | letor_db);
-    if(mset.empty())
+    if( mset.empty())
```

Comment:
Still wrong - the space goes before the `(`.  Running `xapian-check-patch` before you push would save you a lot of time with this sort of thing.